skin/src/vue/c-content.vue

```diff
@@ -114,14 +114,22 @@ onUpdated(() => {
     </template>
     <template v-else>
       <!-- eslint-disable vue/no-v-html -->
-      <XBreadcrumb></XBreadcrumb>
+      <div class="breadcrumb">
+        <XBreadcrumb></XBreadcrumb>
+      </div>
```

Comment:
Note, another options is:
```suggestion
<XBreadcrumb class="breadcrumb"></XBreadcrumb>
```

And to change the css to :
```css
.breadcrumb {
  display: block;
  padding: 8px;
}
```

Reply:
See https://vuejs.org/guide/essentials/class-and-style.html for some explanations regarding how class/style works with components.

Reply:
ok, we can keep it like that for now.